jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/DateTimePickersTests.java

```diff
@@ -1,62 +1,102 @@
+    @Test
+    public void pickDateViaDialogTextFieldTest(){
+        dialogPicker.iCore().label().has().text("Date picker dialog");
+        dialogPicker.value().setText("10/10/2021");
+        dialogPicker.has().text("10/10/2021");
+    }
 
-        dateTimePickersFrame.dateTimePickersInput.setValue("10/10/2021");
-        dateTimePickersFrame.dateTimePickersInput.is().valueContains("10/10/2021");
+    @Test
+    public void pickTimeViaTimePopUpTest(){
+        timePicker.expand();
+        timePicker.list().get("5").click(1, 1);
+        timePicker.list().get("15").click(3, 3);
+        timePicker.command("ENTER");
+        timePicker.has().text("05:14 PM");
     }
 
     @Test
-    public void timePicker(){
-        timePickerPage.open();
+    public void pickTimeViaTimeTextFieldTest(){
+        timePicker.iCore().label().has().text("Time picker");
+        timePicker.value().setText("05:14 AM");
+        timePicker.has().text("05:14 AM");
+    }
 
-        String currentTime = dateTimePickersFrame.dateTimePickersInput.getValue();
-        dateTimePickersFrame.dateTimePickersIcon.toggle();
-        dateTimePickersFrame.dateTimePickersCancelButton.toggle();
-        dateTimePickersFrame.dateTimePickersInput.is().valueContains(currentTime);
+    @Test
+    public void escapeTimePickingUsingTimePopUpTest(){
+        timePicker.expand();
+        timePicker.list().get("5").click(1, 1);
+        timePicker.list().get("15").click(3, 3);
+        timePicker.command("ESCAPE");
+        timePicker.has().text("09:11 PM");
+    }
 
-        dateTimePickersFrame.dateTimePickersIcon.is().visible();
-        dateTimePickersFrame.dateTimePickersIcon.toggle();
-        dateTimePickersFrame.timeTen.is().visible();
-        dateTimePickersFrame.timeTen.toogleXY();
-        dateTimePickersFrame.dateTimePickersOkButton.toggle();
-        dateTimePickersFrame.dateTimePickersInput.is().valueContains("10:");
+    @Test
+    public void pickDateViaBirthdayPickerTextField(){
+        birthdayPicker.iCore().label().has().text("Birthday");
+        birthdayPicker.has().text("2017-05-24");
+        birthdayPicker.value().setText("2021-04-05");
+        birthdayPicker.has().text("2021-04-05");
+    }
 
-        dateTimePickersFrame.dateTimePickersInput.setValue("10:10");
-        dateTimePickersFrame.dateTimePickersInput.is().valueContains("10:10");
+    @Test
+    public void pickAppointmentViaAppointmentPickerTextField(){
+        appointmentPicker.iCore().label().has().text("Next appointment");
+        appointmentPicker.has().text("2017-05-24T10:30");
+        appointmentPicker.value().setText("2021-11-12T11:12");
+        appointmentPicker.has().text("2021-11-12T11:12");
     }
 
+    @Test
+    public void pickTimeViaAlarmPickerTextField(){
+        alarmPicker.iCore().label().has().text("Alarm clock");
+        alarmPicker.has().text("07:30");
+        alarmPicker.value().setText("08:00");
+        alarmPicker.has().text("08:00");
+    }
```

Comment:
Please group tests only by picker, code to be more readable. Like "datePickerInlineTest", "datePickerDialogTest" and etc.